src/views/waterUse/WaterUse.vue

```diff
@@ -242,22 +273,22 @@
     margin: 4px 1px 0 0;
   }
 }
-path.wu-dots{
-  stroke: #0080FF;
+.wu-dots{
+  stroke: $thermo;
   stroke-linecap: round;
   opacity: 0.6;
 }
 #waterUseBarChartContainer{
   .wu-bars-hover{
     fill-opacity: 0;
     &:hover, &:focus{
-      fill: yellow;
+      fill: rebeccapurple;
```

Comment:
lol, is that right?

Reply:
It is, rebecca gets her own purple I guess.